Dockerfile

```diff
@@ -16,15 +16,21 @@ RUN apk add --no-cache \
   tzdata \
   && rm -rf /var/cache/apk/*
 
-RUN mkdir -p /app
+RUN mkdir -p /app /bin /app/config
 WORKDIR /app
 
 RUN gem update --system 2.6.10
 
+ADD https://github.com/bundler/bundler-api/raw/master/versions.list /app/config/versions.list
```

Comment:
I think instead of this (that file is really old now) you probably wan to do something like `RUN curl -o /app/config/versions.list https://rubygems.org/versions`?

Reply:
Hmmm this is exactly what we use in production today